lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -269,8 +269,8 @@ class IndexQueryTestCase(ModuleStoreTestCase):
     NUM_PROBLEMS = 20
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 10, 175),
-        (ModuleStoreEnum.Type.split, 4, 171),
+        (ModuleStoreEnum.Type.mongo, 10, 178),
+        (ModuleStoreEnum.Type.split, 4, 174),
```

Comment:
Do we know why these are going up?

Reply:
i'm guessing one is from the content type gating check in this function https://github.com/edx/edx-platform/pull/26143/files#diff-20826fcfc64f6ff8931e4b2e686a9a5a588b59c2eb808bb0eee0ed3ae4efa575R292
and maybe the has access check inside the service check in that function. 

though not really sure, do you know if there is an easy way to identify the new queries in cases like this where there's a long list?
i guess I can print out the queries on both my branch and master, then write a little script to diff them?